test/detection/test_vlm.py

```diff
@@ -359,19 +359,140 @@ def test_from_qwen_2_5_vl(
+        ),
+        (
+            does_not_raise(),
+            "```json\n[]\n```",
+            (1000, 1000),
+            None,
+            (np.empty((0, 4)), None, np.empty(0, dtype=str)),
+        ),
+        (
+            does_not_raise(),
+            """```json
+            [
+                {"box_2d": [100, 200, 300, 400], "label": "cat"}
+            ]
+            ```""",
+            (1000, 500),
+            None,
+            (
+                np.array([[200.0, 50.0, 400.0, 150.0]]),
+                None,
+                np.array(["cat"], dtype=str),
+            ),
+        ),
+        (
+            does_not_raise(),
+            """```json
+            [
+                {"box_2d": [10, 20, 110, 120], "label": "cat"},
+                {"box_2d": [50, 100, 150, 200], "label": "dog"}
+            ]
+            ```""",
+            (640, 480),
+            None,
+            (
+                np.array([[12.8, 4.8, 76.8, 52.8], [64.0, 24.0, 128.0, 72.0]]),
+                None,
+                np.array(["cat", "dog"], dtype=str),
+            ),
+        ),
+        (
+            does_not_raise(),
+            """```json
+            [
+                {"box_2d": [10, 20, 110, 120], "label": "cat"}
+            ]
+            ```""",
+            (640, 480),
+            ["dog", "person"],
+            (np.empty((0, 4)), np.empty(0, dtype=int), np.empty(0, dtype=str)),
+        ),
+        (
+            does_not_raise(),
+            """```json
+            [
+                {"box_2d": [10, 20, 110, 120], "label": "cat"},
+                {"box_2d": [50, 100, 150, 200], "label": "dog"}
+            ]
+            ```""",
+            (640, 480),
+            ["person", "dog"],
+            (
+                np.array([[64.0, 24.0, 128.0, 72.0]]),
+                np.array([1]),
+                np.array(["dog"], dtype=str),
+            ),
+        ),
+        (
+            does_not_raise(),
+            """```json
+            [
+                {"box_2d": [10, 20, 110, 120], "label": "cat"},
+                {"box_2d": [50, 100, 150, 200], "label": "dog"}
+            ]
+            ```""",
+            (640, 480),
+            ["cat", "dog"],
+            (
+                np.array([[12.8, 4.8, 76.8, 52.8], [64.0, 24.0, 128.0, 72.0]]),
+                np.array([0, 1]),
+                np.array(["cat", "dog"]),
+            ),
+        ),
+        (
+            pytest.raises(ValueError),
+            """```json
+            [
+                {"box_2d": [10, 20, 110, 120], "label": "cat"}
+            ]
+            ```""",
+            (0, 480),
+            None,
+            None,
+        ),
+        (
+            pytest.raises(ValueError),
+            """```json
+            [
+                {"box_2d": [10, 20, 110, 120], "label": "cat"}
+            ]
+            ```""",
+            (640, -100),
+            None,
+            None,
+        ),
+    ],
+)
```

Comment:
Please add a one-sentence description to each test case explaining its purpose, following the style used in `test_from_paligemma`. Each description should clearly state what is being tested in that specific case.